src/backend/api/Fusion.Resources.Api/Notifications/RequestsNotificationHandler.cs

```diff
@@ -17,59 +16,18 @@
 namespace Fusion.Resources.Api.Notifications
 {
     public class RequestsNotificationHandler :
-        INotificationHandler<RequestCreated>,
-        INotificationHandler<RequestApprovedByCompany>,
-        INotificationHandler<RequestApprovedByContractor>,
         INotificationHandler<RequestDeclinedByCompany>,
         INotificationHandler<RequestDeclinedByContractor>
     {
         private readonly IMediator mediator;
         private readonly IFusionNotificationClient notificationClient;
         private readonly IProjectOrgResolver orgResolver;
-        private readonly IUrlResolver urlResolver;
 
-        public RequestsNotificationHandler(IMediator mediator, IFusionNotificationClient notificationClient, IProjectOrgResolver orgResolver, IUrlResolver urlResolver)
+        public RequestsNotificationHandler(IMediator mediator, IFusionNotificationClient notificationClient, IProjectOrgResolver orgResolver)
         {
             this.mediator = mediator;
             this.notificationClient = notificationClient;
             this.orgResolver = orgResolver;
-            this.urlResolver = urlResolver;
-        }
-
-        public async Task Handle(RequestCreated notification, CancellationToken cancellationToken)
-        {
-            var request = await GetRequestAsync(notification.RequestId);
-            var recipients = await CalculateExternalCRRecipientsAsync(request);
-            var requestsUrl = await urlResolver.ResolveActiveRequests(request.Project.OrgProjectId, request.Contract.OrgContractId);
```

Comment:
Not that it matters, but for future reference. I don't think this will work, as it would not really give any benefit for the user to get a url to the api endpoint? 

Should always think of routing the user to the portal and app.

Reply:
Ok, this worked without issue by using the **referer** on the request in the Url resolver. 
Anyway, we are using the portal address in Functions.